src/components/filters/component.js

```diff
@@ -1,12 +1,22 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import { replace } from 'redux-first-router';
+import queryString from 'query-string';
+import isArray from 'lodash/isArray';
 import Button from 'components/button';
 import Modal from 'components/modal';
 import initialState from 'modules/filters/initial-state';
 import { filtersData } from './constants';
 
-const Filters = ({ filters, resetFilters, setFilter }) => {
-  const [filtersResult, setFiltersResult] = useState(filters);
+const Filters = ({ location, filters, resetFilters, setFilter }) => {
+  const { pathname, query } = location;
+  const queryFilters = {};
```

Comment:
where is query coming from?
I get it as undefined so it throws the following error: "Cannot convert undefined or null to object"

Reply:
It's solve, could you check?